.github/workflows/selftest.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   selftest:
     name: Self-test
     runs-on: ubuntu-latest
-    container: quay.io/ovirt/buildcontainer:stream8
+    container: quay.io/ovirt/buildcontainer:el8stream
```

Comment:
So the container image has been renamed as well? Also, I think it would be good to have a matrix with el9stream too in the self test so we can check both cases - that's the main use case.

Reply:
Yeah the container was renamed to align with OST, I will also add the el9 to test both cases.